### ISSUE TITLE
Signing Certificate Expired

### ISSUE THREAD

**Issue by Saberpeep** [bluskript/xsoverlay-notifier#8]
I can't get this to install. I have set my execution policy to allow the ps1 script to run. The ps1 install script fails with the following error: 
```
Add-AppxPackage: Deployment failed with HRESULT: 0x80073CF0, Package could not be opened.

Common::Deployment::MsixvcStagingSession::GetManifestReader in MsixvcStagingSession failed with error 0x80070570.
```

I have also tried manually installing the cert as outlined in the install script, and I can see it installed correctly in certlm, and tried running the msix manually but it still complains about the certificate. Are you sure you have included a valid cert with the release?

Running unmodified Win 11 Home 23H2

**Comment by Saberpeep** [comment]
Oh lol yeah the issue is super obvious, the cert literally says it expired 1st January 2024 right on it. Updating the issue title to match...

**Comment by Saberpeep** [comment]
Oh huh well it let me install anyways if I just pass the `-AllowUnsigned` flag to `Add-AppPackage`. Not sure of the security implications of that, but I suppose I am installing a mysterious app from github in the first place.

```
Add-AppPackage .\XSOverlayNotifier.msix -AllowUnsigned
```

**Comment by bluskript** [comment]
> Oh huh well it let me install anyways if I just pass the `-AllowUnsigned` flag to `Add-AppPackage`. Not sure of the security implications of that, but I suppose I am installing a mysterious app from github in the first place.
> 
> ```
> Add-AppPackage .\XSOverlayNotifier.msix -AllowUnsigned
> ```

wait there was an AllowUnsigned flag this entire time?? I can't believe I missed that... if that's the case then I can get rid of the certificate process completely, thanks for mentioning this

**Comment by Saberpeep** [comment]
Yeah I'm not really familiar with all this, I've only had to deal with these msix packages before when trying to install K2VR Amethyst dev builds. I just googled "install msix without certificate" and I came across this page: 
https://learn.microsoft.com/en-us/windows/msix/package/unsigned-package 
which shows the `-AllowUnsigned` argument.

**Comment by Krzeszny** [comment]
Gotta love how you need to be a programmer to install an app, even though this isn't even Linux...

Well, I'm not a programmer.
```
PS C:\Users\krzes\Downloads> Add-AppPackage .\XSOverlayNotifier.msix -AllowUnsigned
Add-AppxPackage : A parameter cannot be found that matches parameter name 'AllowUnsigned'.
```

**Comment by Saberpeep** [comment]
> Gotta love how you need to be a programmer to install an app, even though this isn't even Linux...
> 
> Well, I'm not a programmer.
> 
> ```
> PS C:\Users\krzes\Downloads> Add-AppPackage .\XSOverlayNotifier.msix -AllowUnsigned
> Add-AppxPackage : A parameter cannot be found that matches parameter name 'AllowUnsigned'.
> ```

Uhh so upon further research it seems the allowUnsigned parameter was added in Windows 11 😅

**Comment by Erallie** [comment]
> > Oh huh well it let me install anyways if I just pass the `-AllowUnsigned` flag to `Add-AppPackage`. Not sure of the security implications of that, but I suppose I am installing a mysterious app from github in the first place.
> > ```
> > Add-AppPackage .\XSOverlayNotifier.msix -AllowUnsigned
> > ```
> 
> wait there was an AllowUnsigned flag this entire time?? I can't believe I missed that... if that's the case then I can get rid of the certificate process completely, thanks for mentioning this

I can confirm this works on Windows 11. Not sure about Windows 10 though.
